src/ansys/dynamicreporting/core/adr_report.py

```diff
@@ -239,7 +239,7 @@ def export_pdf(
         if self.service.serverobj is None:  # pragma: no cover
             self.service.logger.error("No connection to any server")
             return ""
-        try:
+        try:  # pragma: no cover

```

Comment:
Currently this works only if Qt is available, so only for local installations. Skip for now.